go/oasis-test-runner/rust/builder.go

```diff
@@ -82,6 +82,7 @@ func (b *Builder) Build(pkg string) error {
 	}
 
 	cmd.Args = append(cmd.Args, "--package", pkg)
+	cmd.Args = append(cmd.Args, "--release")
```

Comment:
The `elf2sgxs` command also needs this flag.

Reply:
Yes, figured that out because tests are failing.